extension/js/common/platform/util.ts

```diff
@@ -32,3 +34,11 @@ export const moveElementInArray = <T>(arr: Array<T>, oldIndex: number, newIndex:
   arr.splice(newIndex, 0, arr.splice(oldIndex, 1)[0]);
   return arr;
 };
+
+export const iso2022jpToUtf = (content: Buf) => {
+  if (!TextDecoder) {
```

Comment:
You have edited `extension/js/common/platform/util.ts` which runs in the browser. My earlier comment was about `test/source/platform/util.ts` which runs in node.

That's why they are under `platform` folder, the files are **different** on each platform.

Therefore, the error message is appropriate for the Node implementation (where it should be left unimplemented and not throw).

For browser, you would say `Your browser does is not supported (missing TextDecoder)`.

Reply:
Oh, sorry, I thought that they are the same files. 
I will change that.
Thanks